src/main/java/core/basesyntax/exceptions/IncorrectFileNameException.java

```
@@ -0,0 +1,7 @@
+package core.basesyntax.exeptions;
+
+public class IncorrectFileNameException extends RuntimeException {
+    public IncorrectFileNameException(String message) {
+        super(message);
+    }
```

This exception class provides only a (String) constructor. To follow the checklist guidance "Don't ignore exceptions in try-catch blocks... Let's rethrow a RuntimeException with an **informative** message and exception object," add an overloaded constructor that accepts (String message, Throwable cause) and calls `super(message, cause)`. This will let higher-level code preserve the original cause when wrapping exceptions.